src/main/java/org/humancellatlas/ingest/file/FileArchiveResult.java

```diff
@@ -0,0 +1,17 @@
+package org.humancellatlas.ingest.file;
+
+import java.time.Instant;
+import lombok.Data;
+
+@Data
+public class FileArchiveResult {
+    private Instant lastArchived;
+    private Boolean compressed;
+    private String md5;
+    private String enaUploadPath;
+    private Boolean success;
```

Comment:
I think this is redundant with the error attribute. It's probably better to just retain the error attribute and remove this. If there's no error, that means archiving is successful.